src/actions/accommodationActions.js

```diff
@@ -136,3 +142,18 @@ export const getDeactivatedAccommodation = () => (dispatch) => backendCall.get('
       accommodationType(DEACTIVATED_ACCOMMODATION_ERROR, error.response.data),
     );
   });
+
+export const activatedAccommodation = (slug, reasons) => (dispatch) => backendCall.patch(`/accommodations/activate/${slug}`, { reasons }, { headers: activationHeaders })
```

Comment:
I suggest you rename the action to `activateAccommodation` since it is an action it should be a verb.